kubejs/data/create/recipes/cutting/compat/byg/mangrove_log.json

```diff
@@ -0,0 +1,19 @@
+{
+  "type": "create:cutting",
+  "conditions": [
```

Comment:
Why are all these cutting recipes here?

Reply:
This was part of my fixing parse recipe errors we had 😅 Except the fact I did few PRs to mods with those jsons I have no idea why I included them instead of just doing them in the script >.>
Gonna remake them to the script version 👍

Reply:
Okay so...
a) I answered this when I just woke up 😅 So I screwed up explaining what is wrong.
Create has BYG compat built in, however, they didn't remove the Mangrove Wood from it, when BYG Did remove it from the game, generating Recipe Parse Errors. This is a fix for that on our end, and PR on Create github is live :D
https://github.com/Creators-of-Create/Create/pull/4630

b) As it turns out, I can't remove the errors without overriding the json >.> So its either I do scripts for injecting datapacks, or leave this alone with just condition forge:false until Create merges my PR and updates the mod ^^"
Same problem is with other recipes I fixed this way, they are probably not possible to override by recipe event, but it might be possible to override by Datapack event script, but I think just leaving this as json should be fine?